addons/binding/org.openhab.binding.hdanywhere/ESH-INF/thing/matrix.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="hdanywhere"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- HDanywhere Thing Type -->
+    <thing-type id="matrix">
```

Comment:
How many variants are there? Would it be a lot of effort to define the different products directly? You define 8 channels, but some devices have only 4 of them, right? So this feels a bit awkward.
Also, if the number of input ports is known, it could be defined in the state description in order to list the valid options.